extensions/spring-xd-extension-reactor/src/main/java/org/springframework/xd/integration/reactor/config/SyslogInboundChannelAdapterParser.java

```diff
@@ -21,28 +21,33 @@
 import org.springframework.beans.factory.xml.ParserContext;
 import org.springframework.integration.config.xml.AbstractChannelAdapterParser;
 import org.springframework.integration.config.xml.IntegrationNamespaceUtils;
+import org.springframework.xd.integration.reactor.syslog.SyslogInboundChannelAdapterConfiguration;
 import org.springframework.xd.integration.reactor.syslog.SyslogInboundChannelAdapter;
 import org.w3c.dom.Element;
+import reactor.io.codec.DelimitedCodec;
+import reactor.io.codec.StandardCodecs;
 
 /**
  * @author Jon Brisbin
+ * @author Stephane Maldini
  */
 public class SyslogInboundChannelAdapterParser extends AbstractChannelAdapterParser {
 
 	@Override
 	protected AbstractBeanDefinition doParse(Element element, ParserContext parserContext, String channelName) {
 		BeanDefinitionBuilder builder = ReactorNamespaceUtils.createBeanDefinitionBuilder(
-				SyslogInboundChannelAdapter.class,
+				SyslogInboundChannelAdapterConfiguration.class,
 				element
 				);
 
 		IntegrationNamespaceUtils.setValueIfAttributeDefined(builder, element, "host", "host");
 		IntegrationNamespaceUtils.setValueIfAttributeDefined(builder, element, "port", "port");
 
-		IntegrationNamespaceUtils.setReferenceIfAttributeDefined(builder, element, "channel", "outputChannel");
```

Comment:
The whole thing blows up even earlier since `ReactorNamespaceUtils.createBeanDefinitionBuilder` is trying to set the reactor environment on a ctor for `SyslogInboundChannelAdapterConfiguration` which doesn't exist.